bokehjs/src/vendor/slick-grid-2.1.0/slick-default-theme.css

```diff
@@ -96,7 +96,6 @@ classes should alter those!
 
 .bk-slick-row.loading {
   opacity: 0.5;
-  filter: alpha(opacity = 50);
```

Comment:
This is for pre-IE8. Hoping we can live without it, as LESS does not support it.